utils_nlp/models/xlnet/utils.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+from matplotlib import pyplot
+from sklearn.metrics import confusion_matrix
+import seaborn as sn
+import numpy as np
+import pandas as pd
+def generate_confusion_matrix(y_true, y_pred,labels):
+    """Function that prints out a graphical representation of confusion matrix using Seaborn Heatmap
+
+    Args:
+        y_true: True labels from dataset
+        y_pred: Predicted labels from the models
+        labels: A list of labels
+
+    """
+    conf_matrix=np.array(confusion_matrix(y_true, y_pred))
+    conf_matrix=np.divide(conf_matrix.astype('float'), conf_matrix.sum())
+    conf_dataframe = pd.DataFrame(conf_matrix,labels,labels)
+    dimensions_plot = (8, 5)
```

Comment:
it would be good to have these parameters as input variables, defaulting to what you want